### PR TITLE
Update Swedish, Finnish and English translations

### PR DESCRIPTION
**Pull request by AnnaSmaros**

### CODE REVIEW

File: languages/en.ini
Comment (demiankatz) [inline languages/en.ini]: In #2020, we addressed a long standing inconsistency where some lines referred to "favorites" and others referred to "saved items" in favor of using "saved items" everywhere. I think it would probably be better not to reintroduce that inconsistency in just one line. If you think it would be better to change "resources" to "items" for consistency with other language, I would not object to that adjustment!

(I also wouldn't close the door on a conversation about bringing back "favorites" but I think if we want to use that term, we need to make a lot more changes than just one line).

In any case, thanks for the various improvements here; I just think we should sort out this detail before moving forward with a merge!

**Review by demiankatz — CHANGES_REQUESTED** [review]

File: languages/en.ini
Comment (AnnaSmaros) [reply languages/en.ini]: Thank you for your feedback. I was not aware of this earlier conversation, but had suspicions about "favorites" not being widely used.
I will amend my request. I think "items" instead of "resources" is an acceptable solution.

**Review by demiankatz — APPROVED** [review]
Thanks, @AnnaSmaros, the English translations look consistent to me now!

I notice that the Swedish translation already uses "favoriter" widely, so I trust your judgment on the use of the word there.

Just checking for a second opinion from @EreMaijala on the Finnish before merging this, since I'm not really qualified to look at that myself. :-)

**Review by EreMaijala — APPROVED** [review]
Looks good to me!

**Comment by demiankatz** [comment]
Backported to release-10.2.

**Pull request merged**
